README.md

```diff
@@ -40,4 +45,13 @@ will result in
 
 ```
 
+Username and password should be set at `/ft/_credentials/vulcand/` like it was mentioned above, to be able to set for the marked services.
 These routing rules will change as we develop. The idea is they are in a single place in this application, not spread out across many unmaintainable sidekick services.
+
+## Test the app locally
+
+1. Install [__etcd__](https://github.com/coreos/etcd)
+2. `go get github.com/Financial-Times/vulcan-config-builder`
+3. `git clone https://github.com/Financial-Times/vulcan-config-builder.git && cd vulcan-config-builder`
```

Comment:
Just for clarity: you don't actually need both 'go get' and 'git clone'. Running 'go build' is also redundant here.